extensions/binding/org.eclipse.smarthome.binding.iota/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: ., lib/jota-0.9.11-SNAPSHOT-jar-with-dependencies.jar
```

Comment:
right now the `jota-0.9.11-SNAPSHOT-jar-with-dependencies.jar` seems to be unused in the binding. I think this is a good thing since the IO bundle should encapsulate all the iota specific communication. But the lib can be removed here.

Reply:
Removed